src/main/resources/org/primefaces/Messages_de.properties

```diff
@@ -30,6 +31,14 @@ primefaces.panel.aria.OPTIONS_MENU = Optionsmenü umschalten
 primefaces.panel.aria.TOGGLE = Bedienfeld ein- und ausblenden
 primefaces.password.INVALID_MATCH = {0}: Validierungsfehler.
 primefaces.password.INVALID_MATCH_detail = {0} sollte mit {1} übereinstimmen.
+primefaces.picklist.ADD = Hinzufügen
+primefaces.picklist.ADD_ALL = Alle hinzufügen
+primefaces.picklist.REMOVE = Entfernen
+primefaces.picklist.REMOVE_ALL = Alle entfernen
+primefaces.picklist.MOVE_UP = Nach oben verschieben
+primefaces.picklist.MOVE_TOP = Nach oben verschieben
```

Comment:
Why is the text the same for both?

Reply:
My bad. It was way too late yesterday. Thanks for spotting!

Reply:
I resolved the translation and added your NL translation as well.